src/CMakeLists.txt

```diff
@@ -416,9 +420,13 @@ add_executable(mudlet ${mudlet_SRCS} ${mudlet_RCCS} ${mudlet_HDRS} ${mudlet_UIS}
 # https://github.com/Mudlet/Mudlet/issues/6314
 #
 # * Produce a timestamped message on the OS's command line when an autosave of
-# the map file gets requested and when such a request gets actioned (or
-# rejected because the profile is being loaded):
+#   the map file gets requested and when such a request gets actioned (or
+#   rejected because the profile is being loaded):
 # target_compile_definitions(mudlet PRIVATE DEBUG_MAPAUTOSAVE)
+#
+# * Produce all the time the surprise that normally will only occur on the
+#   first day of the fourth month of the Gregorian calendar year:
+target_compile_definitions(mudlet PRIVATE DEBUG_EASTER_EGGS)
```

Comment:
```suggestion
// target_compile_definitions(mudlet PRIVATE DEBUG_EASTER_EGGS)
```

Reply:
This isn't a C++ file - you need to use `#` not `//`...!